kolibri/core/assets/src/state/store.js

```diff
@@ -45,6 +45,7 @@ export const initialState = {
     facilityConfig: {},
     facilities: [],
     connection: baseConnectionState,
+    currentSnackbar: null,
```

Comment:
Ah, ok - so it's more of a connection state alert, rather than connection state. Could rename the constants slightly to reflect that?